bitnami/influxdb/templates/application.yaml

```diff
@@ -0,0 +1,323 @@
+        {{- if and .Values.defaultInitContainers.volumePermissions.enabled .Values.persistence.enabled (eq .Values.objectStore "file") }}
+        {{- include "influxdb.defaultInitContainers.volumePermissions" . | nindent 8 }}
+        {{- end }}
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" .) | trim | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: influxdb
+          image: {{ include "influxdb.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.containerSecurityContext "context" .) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" .) | nindent 12 }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" .) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.command "context" .) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" .) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" .) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.image.debug .Values.diagnosticMode.enabled) | quote }}
+            - name: INFLUXDB_OBJECT_STORE
+              value: {{ .Values.objectStore | quote }}
+            {{- if ne .Values.objectStore "memory" }}
+            - name: INFLUXDB_DATA_DIR
+              value: {{ printf "%s/data" .Values.persistence.mountPath | quote }}
+            {{- end }}
+            - name: INFLUXDB_NODE_ID
+              {{- if eq .Values.objectStore "file" }}
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+              {{- else }}
+              value: {{ .Values.nodeId | quote }}
+              {{- end }}
+            - name: INFLUXDB_CREATE_ADMIN_TOKEN
+              value: {{ ternary "yes" "no" (and .Values.auth.createAdminToken (ne .Values.objectStore "memory")) | quote }}
+            {{- if and .Values.databases .Values.auth.createAdminToken (ne .Values.objectStore "memory") }}
+            - name: INFLUXDB_DATABASES
+              value: {{ .Values.databases | quote }}
+            {{- end }}
+            {{- if or (eq .Values.objectStore "s3") (eq .Values.objectStore "google") (eq .Values.objectStore "azure") }}
```

Comment:
I see that it is possible to add TLS support. We may want to add it in order to comply with checklists

https://github.com/influxdata/influxdb/blob/bf83e7fbb397e579cbee0ca87c5b797fc2ecdcf5/influxdb3/src/help/serve.txt#L30

Reply:
It seems the docs was not updated but it's there https://github.com/influxdata/influxdb/issues/25774

Reply:
Done at https://github.com/bitnami/charts/pull/33743/commits/d6e692d0610b81b5e78e503db4ff471fea734331